### PR TITLE
Fix dApp swiper configuration

### PR DESCRIPTION
**Pull request by bobo-k2**
**Pull Request Summary**

Fix for misconfigured swiper in voting component's dApps list 
Fix for sorting dApps by claimed rewards
Fix for TVL ratio in period stats component

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

### CODE REVIEW

File: src/staking-v3/components/PeriodStats.vue [hunk 1]
```diff
@@ -99,7 +99,7 @@ export default defineComponent({
         .map((x) => ({
           name: x.name,
           iconUrl: x.iconUrl,
-          amount: x.rewardAmount,
+          amount: BigInt(x.rewardAmount),
```
Comment (gtg7784) [inline src/staking-v3/components/PeriodStats.vue:102]:
```
export type PeriodData = {
  dappAddress: string;
  rewardAmount: bigint;
  stakeAmount: bigint;
};
```

I think it's already bigint

**Review by Kahonnohak — APPROVED** [review]
Thank you for quick fix!

File: src/staking-v3/components/PeriodStats.vue (hunk 1 above)
Comment (bobo-k2) [reply src/staking-v3/components/PeriodStats.vue:102]: Me too, but it is treated as string, most likely the value comes as string from API. I will check and cast immediately after fetching.

Comment (bobo-k2) [reply src/staking-v3/components/PeriodStats.vue]: Fixed in https://github.com/AstarNetwork/astar-apps/pull/1317/commits/cdf64d0e6c25fc2a354ec58797e83e6290026ac4 
coverting to big int as soon as data is received

Comment (gtg7784) [reply src/staking-v3/components/PeriodStats.vue]: ah I see, thanks!

**Pull request merged**